deploy/chart/templates/0000_50_olm_03-clusterserviceversion.crd.yaml

```diff
@@ -823,6 +824,8 @@ spec:
                                     description: 'Standard object''s metadata. More
                                       info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata'
                                     type: object
+                                    # TODO: Autogen this...
+                                    x-kubernetes-preserve-unknown-fields: true
```

Comment:
won't this disappear if we regen?

Reply:
That is correct.

The deployment template's metatdata isn't marked as something that should not be pruned - as such so you can't set `metadata.annotations.label` unless this is added. I could use yq to autogen this as part of the command.

Reply:
The `yq` command to update the crd has been added.